ape_etherscan/client.py

```diff
@@ -122,6 +123,7 @@ def __authorize(self, params_or_data: Optional[Dict] = None) -> Optional[Dict]:
         api_key = os.environ.get(env_var_key)
         if api_key and (not params_or_data or "apikey" not in params_or_data):
             params_or_data = params_or_data or {}
+            api_key = random.choice(api_key.split(","))
```

Comment:
consideration: we could also call `strip()` in case the user does a value like `"KEY_0, KEY_1"` (comma + space sep)